pkgs/applications/editors/jetbrains/source/build.nix

```diff
@@ -226,6 +241,9 @@ stdenvNoCC.mkDerivation rec {
     sed \
       -e 's|BROKEN_PLUGINS_HERE|${./brokenPlugins.json}|' \
       -i platform/build-scripts/src/org/jetbrains/intellij/build/impl/brokenPlugins.kt
+    sed \
+      -e 's|XPLAT_LAUNCHER_PREBUILT_PATH_HERE|${xplat-launcher}|' \
```

Comment:
We should really use substituteInPlace here if no regex's are used to not when those no longer match.

Reply:
All of `sed`s are replaced with `substituteInPlace`.